src/ch05-02-example-structs.md

```diff
@@ -225,42 +242,44 @@ struct Rectangle {
 fn main() {
     let rect1 = Rectangle { width: 30, height: 50 };
 
-    println!("rect1 is {:?}", rect1);
+    println!("rect1 to {}", rect1);
 }
 ```
 
-<span class="caption">Listing 5-12: Adding the annotation to derive the `Debug`
-trait and printing the `Rectangle` instance using debug formatting</span>
+<span class="caption">Listing 5-12: Dodanie adnotacji derywacji cechy `Debug`
+i wyświetlanie instancji `Rectangle` formatowaniem przeznaczonym do celów debugowania</span> 
 
-Now when we run the program, we won’t get any errors, and we’ll see the
-following output:
+Teraz kiedy uruchomimy program nie wyskoczy nam żaden błąd, a naszym oczom
+ukaże się poniższy tekst:
 
 ```text
-rect1 is Rectangle { width: 30, height: 50 }
+rect1 to Rectangle { width: 30, height: 50 }
 ```
 
-Nice! It’s not the prettiest output, but it shows the values of all the fields
-for this instance, which would definitely help during debugging. When we have
-larger structs, it’s useful to have output that’s a bit easier to read; in
-those cases, we can use `{:#?}` instead of `{:?}` in the `println!` string.
-When we use the `{:#?}` style in the example, the output will look like this:
+No nieźle! Nie jest to może najpiękniejsza reprezentacja, ale spełnia swoje zadanie i
+pokazuje wartości wszystkich atrybutów tej instancji,
```

Comment:
atrybutów -> pól

Reply:
@katecpp jeśli też się zgadzasz z @beling to pronowałbym na rozwiązanie: stworzę nowy PR po zmergowaniu tego, w którym zmienię także 05-01, bo jeśli się nie mylę tam również używałem `atrybutu` a nie `pola`; ergonomiczniej w historii gita zawarta byłaby zmiana z `atrybutu` na `pole`

Reply:
wydaje mi się, że w stosunku do struktur i klas używa się słowa pola, we wszystkich językach

Reply:
zmieniłem